hibernate-core/src/main/java/org/hibernate/dialect/DerbyDialect.java

```diff
@@ -438,33 +436,22 @@ public String timestampdiffPattern(TemporalUnit unit, TemporalType fromTemporalT
 
 	@Override
 	public void appendBooleanValueString(SqlAppender appender, boolean bool) {
-		if ( getVersion().isBefore( 10, 7 ) ) {
-			appender.appendSql( bool ? '1' : '0' );
-		}
-		else {
-			appender.appendSql( bool );
-		}
+		appender.appendSql( bool );
 	}
```

Comment:
I think you can just remove the method as the super implementation will do the right thing

Reply:
Yeah you are right, I misremembered the base implementation